libs/components/src/lib/dial-pad/dial-pad.scss

```
@@ -0,0 +1,40 @@
+@use '../../../../../dist/libs/tokens/scss/tokens.constants' as constants;
+@use '../../../../shared/src/lib/sass/mixins/border-radius' as
+	border-radius-variables;
+
+$inline-size: 230px;
+$gap: 16px;
+
+.base {
+	display: grid;
+	margin: 16px;
+	gap: $gap;
+	inline-size: $inline-size;
+}
+
+.digits {
+	display: grid;
+	gap: $gap;
+	grid-template-columns: repeat(3, 1fr);
+	grid-template-rows: repeat(4, 1fr);
+	inline-size: $inline-size;
+}
+
+.phone-field {
+	--vvd-size-density: 2;
+
+	align-self: flex-start;
+	grid-column: 1 / -1;
+}
+
+.digit-btn {
+	display: flex;
```

```suggestion
--vvd-button-accent-primary: var(#{constants.$vvd-color-neutral-600});

	display: flex;
```
this is for the text of the digits.
but this will also effect the digits - so you'll need to add class for the slotted icon and set it with color:
color: var(#{constants.$vvd-color-canvas-text});